code/modules/urist/items/clothes/head.dm

```diff
@@ -318,6 +318,109 @@ obj/item/clothing/head/princessbow
 	cold_protection = HEAD
 	min_cold_protection_temperature = 243.15
 
+
+
+//Civil War hats
+
+ /obj/item/clothing/head/urist/civilwar/usltgenhat
+	name = "US Army Lieutenant General Hat"
+	desc = "A hat worn by the commander of an army."
+	icon_state = "Lieut General US hat"
+	item_state = "Lieut General US hat"
+
+ /obj/item/clothing/head/urist/civilwar/usbrighat
+	name = "US Army Brigadier General Hat"
+	desc = "A hat worn by the commander of an army."
+	icon_state = "Brig General US hat"
+	item_state = "Brig General US hat"
+
+ /obj/item/clothing/head/urist/civilwar/uscolhat
+	name = "US Army Colonel Hat"
+	desc = "A hat worn by an infantry officer."
+	icon_state = "Colonel of Infantry US hat"
+	item_state = "Colonel of Infantry US hat"
+
+ /obj/item/clothing/head/urist/civilwar/uspvthat
+	name = "US Army Private Hat"
+	desc = "A blue hat worn by an infantryman."
+	icon_state = "Private Infantry US hat"
+	item_state = "Private Infantry US hat"
+
+ /obj/item/clothing/head/urist/civilwar/csapvthat
+	name = "CS Army Private Hat"
+	desc = "A grey hat worn by an infantryman."
+	icon_state = "Private Infantry CS hat"
+	item_state = "Private Infantry CS hat"
+
+ /obj/item/clothing/head/urist/civilwar/csaartycpthat
+	name = "Artillery Captain Hat"
+	desc = "A hat worn by an artillery officer."
+	icon_state = "Captain Artillery CS hat"
+	item_state = "Captain Artillery CS hat"
+
+ /obj/item/clothing/head/urist/civilwar/csagenhat
+	name = "CS Army General Hat"
+	desc = "A hat worn by an army commander."
+	icon_state = "General CS hat"
+	item_state = "General CS hat"
+
+ /obj/item/clothing/head/urist/civilwar/fancyheaddress
+	name = "Fancy Headdress"
+	desc = "Geronimo!"
+	icon_state = "headpiece"
+	item_state = "headpiece"
+
+ /obj/item/clothing/head/urist/civilwar/featherband
+	name = "Feather Headband"
+	desc = "A headband with a single feather on it."
+	icon_state = "single fether"
+	item_state = "single fether"
+
+ /obj/item/clothing/head/urist/civilwar/tricorn
+	name = "Tricorn"
+	desc = "A plain tricorn hat."
+	icon_state = "tricorn_hat"
```

Comment:
alost, the icon_states should be formatted like this. lowercase, underscores instead of spaces.